src/Altinn.Notifications.Persistence/Repository/ApplicationOwnerConfigRepository.cs

```diff
@@ -0,0 +1,81 @@
+﻿using Altinn.Notifications.Core.Models;
+using Altinn.Notifications.Core.Repository.Interfaces;
+using Altinn.Notifications.Persistence.Extensions;
+
+using Npgsql;
+
+using NpgsqlTypes;
+
+namespace Altinn.Notifications.Persistence.Repository;
+
+/// <summary>
+/// Implementation of order repository logic
+/// </summary>
+public sealed class ApplicationOwnerConfigRepository : IApplicationOwnerConfigRepository
+{
+    private const string _selectApplicationOwnerConfigSql =
+        "SELECT _id, orgid, emailaddresses, smsnames " +
+        "FROM notifications.applicationownerconfig " +
+        "WHERE orgid=$1";
+
+    private const string _writeApplicationOwnerConfigSql =
+        "INSERT INTO notifications.applicationownerconfig(orgid, emailaddresses, smsnames) " +
+        "  VALUES ($1, $2, $3) " +
+        "ON CONFLICT (orgid) " +
+        "  DO UPDATE SET emailaddresses = $2, smsnames = $3";
+
+    private readonly NpgsqlDataSource _dataSource;
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="ApplicationOwnerConfigRepository"/> class.
+    /// </summary>
+    /// <param name="dataSource">The npgsql data source.</param>
+    public ApplicationOwnerConfigRepository(NpgsqlDataSource dataSource)
+    {
+        _dataSource = dataSource;
+    }
+
+    /// <inheritdoc/>
+    public async Task<ApplicationOwnerConfig?> GetApplicationOwnerConfig(string orgId)
```

Comment:
I would like to see some caching in front of this method as the data won't change that often we could rather invalidate the cache on insert. ( We will of course get some edge cases for cache across multiple pods, but that is something we can live with for now until we hopefully get a common cache across pods)